mobile_verifier/src/cli/generate.rs

```diff
@@ -0,0 +1,68 @@
+use crate::{
+    env_var,
+    server::{CONNECT_TIMEOUT, DEFAULT_URI, RPC_TIMEOUT},
+    subnetwork_rewards::SubnetworkRewards,
+    Result,
+};
+use chrono::{DateTime, NaiveDateTime, Utc};
+use helium_proto::services::{follower, Endpoint, Uri};
+use poc_store::FileStore;
+
+/// Verify the shares for a given time range
+#[derive(Debug, clap::Args)]
+pub struct Cmd {
+    #[clap(long)]
+    after: NaiveDateTime,
+    #[clap(long)]
+    before: NaiveDateTime,
+    #[clap(long)]
+    input_bucket: String,
+    #[clap(long)]
+    output_bucket: Option<String>,
+    #[clap(long)]
+    dump_to_stdout: bool,
+}
+
+impl Cmd {
+    pub async fn run(self) -> Result {
+        let Self {
+            after,
+            before,
+            input_bucket,
+            output_bucket,
+            dump_to_stdout,
+        } = self;
+
+        tracing::info!(
+            "Verifying shares from bucket {input_bucket} within the following time range: {after} to {before}"
+        );
+
+        let input_store = FileStore::new(None, "us-west-2", input_bucket).await?;
```

Comment:
that region feels like it should be the default value for an optional environment variable

Reply:
can the `FileStore` hide the implementation details of that so we only need to pass it the bucket name perhaps?